test.rb

```diff
@@ -0,0 +1,23 @@
+a = "bbb"
+
+def foo
+    'a'
+end
+
+aaa;
+
+not 1
+
+l = lambda {|a| a }
+l.(x)
+l[x]
+l.call(x)
+
+def b(x)
+  
+end 
+
+
+[a,b,c,d,e]
```

Comment:
__[development][rubocop]__
* Literal `[a,b,c,d,e]` used in void context.

<!-- Comment by [SideCI](https://sideci.com) -->